pygeo/parameterization/DVGeo.py

```diff
@@ -752,6 +752,11 @@ def coordXfer(coords, mode="fwd", applyDisplacement=True, **kwargs):
         # compNames is only needed for DVGeometryMulti, so remove it if passed
         kwargs.pop("compNames", None)
 
+        # check if we want a custom subset of child DVGeos
```

Comment:
This looks like an interesting addition, could you please add a short description of the arg and intended functionality in the docstrings for `addPointSet`?